graph/client/src/app/external-api-impl.ts

```diff
@@ -81,23 +81,13 @@ export class ExternalApiImpl extends ExternalApi {
     }
   }
 
-  async openProjectDetails(projectName: string, targetName?: string) {
-    await this.router.navigate(
-      `/project-details/${encodeURIComponent(projectName)}`
-    );
+  openProjectDetails(projectName: string, targetName?: string) {
+    const pathname = `/project-details/${encodeURIComponent(projectName)}`;
+    const searchParams = new URLSearchParams();
     if (targetName) {
-      this.focusTargetInProjectDetails(targetName);
+      searchParams.set('expanded', targetName);
     }
-  }
-
-  focusTargetInProjectDetails(targetName: string) {
```

Comment:
I know this isn't used anywhere right now, but in the future we will need this for smooth handling in Nx Console. So it would be better if we kept the separate `focusTargetInProjectDetails` method

Reply:
change it back